lib/ast-converter.js

```diff
@@ -215,8 +215,18 @@ function fixExports(node, result, ast) {
         result.range[0] = varToken.getStart();
         result.loc = getLocFor(result.range[0], result.range[1], ast);
 
+        var declarationType = declarationIsDefault ? "ExportDefaultDeclaration" : "ExportNamedDeclaration";
+
+        /**
+         * Prefix exports from TypeScript namespaces with "TS" to distinguish
+         * them from ES2015 exports
+         */
+        if (node.parent && node.parent.kind === SyntaxKind.ModuleBlock) {
```

Comment:
I wonder if we should be more specific with these names. 

TSNamespaceExportDeclaration?